Formula/gucharmap.rb

```diff
@@ -27,7 +27,8 @@ def install
                           "--prefix=#{prefix}",
                           "--disable-Bsymbolic",
                           "--disable-schemas-compile",
-                          "--enable-introspection=no"
+                          "--enable-introspection=no",
+                          "--with-unicode-data=download"
```

Comment:
I could, but there is not really much use to it as the result would be the same.

Also it would make Formula maintenance slightly harder as we would have to update the resources too whenever a new release of `gucharmap` was made...